code/css/theme-sand.css

```diff
@@ -7,11 +9,63 @@ h1 {
   color: #000000;
 }
 
+/* grid layout mobile*/
+
+.rcalc-grid {
+  display: grid;
+  grid-template-columns: 1fr;
+}
+
+.rcalc-log-display {
+  font-size: 20px;
+  overflow-x: auto;
+  overflow-y: auto;
+  margin: auto;
+  background-color: #FFFFFF;
+  width: 268px;
+  height: 392px;
+}
+
+.rcalc-log-display-line {
+  overflow: auto;
+  margin: auto;
+  padding: 0;
+}
+
+.rcalc-container-button-log-clear {
+  text-align: center;
+}
+
+.rcalc-button-log-clear {
+  background-color: #474024;
+  color: #FFFFFF;
+  font-weight: bold;
+  margin-top: 29px;
+  padding: 5px;
+  font-size: 20px;
+  border: none;
+}
+
+/*  box for entire view - needed to position the content */
+
+.rcalc-container-view {
+  text-align: center;
+  background-color: #e2d9b5;
```

Comment:
Just small letters or big letters. Unify it ... #e2d9b5 ... #FFFFFF

Reply:
unified... just small letters used